.github/workflows/docker-publish.yml

```diff
@@ -0,0 +1,47 @@
+---
+# inspired by: 
+# https://docs.github.com/en/actions/publishing-packages/publishing-docker-images and
+# https://github.com/marketplace/actions/docker-metadata-action#basic
+
+name: Publish Docker image
+
+on:
+  push:
+    branches:
+      - 'main'
+    tags:
+      - 'v*'
```

Comment:
I think this needs to be '*.*.*': https://github.com/editorconfig-checker/editorconfig-checker/tags

Reply:
Sorry, the automatic markdown formatting kicked in and I did not looked at my comment after I wrote it.

I mean: `*.*.*`